db/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.6
--- Dumped by pg_dump version 9.5.6
+-- Dumped from database version 9.5.4
+-- Dumped by pg_dump version 9.5.4
```

Comment:
I started adding comments further down this file and noticed a bit too late that your comment was actually pertaining to the whole file. So removed those other comments again and added this small reminder. Sorry for the noise. It's gone again.